bl-languages/fi_FI.json

```diff
@@ -0,0 +1,242 @@
+{
+	"language-data":
+	{
+		"native": "Suomi (Suomi)",
+		"english-name": "Finnish",
+		"last-update": "2016-06-11",
+		"author": "Tuomas K.",
+		"email": "",
+		"website": ""
+	},
+
+	"username": "Käyttäjätunnus",
+	"password": "Salasana",
+	"confirm-password": "Vahvista salasana",
+	"editor": "Kirjoittaja",
+	"dashboard": "Hallintapaneeli",
+	"role": "Tyyppi",
+	"post": "Blogikirjoitus",
+	"posts": "Blogikirjoitukset",
+	"users": "Käyttäjät",
+	"administrator": "Ylläpitäjä",
+	"add": "Lisää",
+	"cancel": "Peruuta",
+	"content": "Sisältö",
+	"title": "Otsikko",
+	"no-parent": "Ei ole",
+	"edit-page": "Muokkaa sivua",
+	"edit-post": "Muokkaa blogikirjoitusta",
+	"add-a-new-user": "Lisää uusi käyttäjä",
+	"parent": "Onko tämä jonkun sivun alasivu?",
+	"friendly-url": "Sivun osoite",
+	"description": "Kuvaus",
+	"posted-by": "Julkaisija",
+	"tags": "Tagit",
+	"position": "Sijainti navigaatiossa",
+	"save": "Tallenna",
+	"draft": "Luonnos",
+	"delete": "Poista",
+	"registered": "Käyttäjä luotu",
+	"notifications": "Ilmoitukset",
+	"profile": "Profiili",
+	"email": "Sähköpostiosoite",
+	"settings": "Asetukset",
+	"general": "Yleiset",
+	"advanced": "Lisäasetukset",
+	"regional": "Alueellinen",
+	"about": "Tietoja",
+	"login": "Kirjaudu sisään",
+	"logout": "Kirjaudu ulos",
+	"manage": "Hallitse",
+	"themes": "Teemat",
+	"prev-page": "Edellinen sivu",
+	"next-page": "Seuraava sivu",
+	"configure-plugin": "Lisäosan asetukset",
+	"confirm-delete-this-action-cannot-be-undone": "Haluatko varmasti poistaa? Tätä toimintoa ei voi perua.",
+	"site-title": "Sivuston otsikko",
+	"site-slogan": "Mainoslause",
+	"site-description": "Sivuston kuvaus",
+	"footer-text": "Alapalkin teksti",
+	"posts-per-page": "Blogikirjoituksia per sivu",
+	"site-url": "Sivuston URL-osoite",
+	"writting-settings": "Kirjoittamisen asetukset",
+	"url-filters": "Osoitteiden asetukset",
+	"page": "Sivu",
+	"pages": "Sivut",
+	"home": "Etusivu",
+	"welcome-back": "Tervetuloa takaisin",
+	"language": "Kieli",
+	"website": "Julkinen sivusto",
+	"timezone": "Aikavyöhyke",
+	"locale": "Maa/Kieli",
+	"new-post": "Uusi blogikirjoitus",
+	"new-page": "Uusi sivu",
+	"html-and-markdown-code-supported": "HTML sekä Markdown toimivat",
+	"manage-posts": "Hallitse blogikirjoituksia",
+	"published-date": "Julkaisuaika",
+	"modified-date": "Muokattu aika",
+	"empty-title": "Tyhjä otsikko",
+	"plugins": "Lisäosat",
```

Comment:
"plugins": "Laajennukset",
Apple uses the word "laajennukset" while "lisäosat" is also correct. I prefer the Apple translation. I have to admit that "laajennukset" is really more like "extensions" while "plugins" is absolutely "lisäosat". This is purely my personal preference.